app/plot_functions.py

```diff
@@ -616,29 +616,34 @@ def create_profile_figure_generation(df_sel: pd.DataFrame) -> go.Figure:
     fig = go.Figure()
 
     add_trace_to_figure(
-        df_sel, fig, component="PV tilted", parameter="Power", fill=True, color="yellow"
+        df_sel,
+        fig,
+        component="PV tilted",
+        parameter="Power",
+        fill=True,
+        color="#FFE000",
     )
     add_trace_to_figure(
         df_sel,
         fig,
         component="Wind onshore",
         parameter="Power",
         fill=True,
-        color="blue",
+        color="#1E78C2",
     )
     add_trace_to_figure(
         df_sel,
         fig,
         component="Wind offshore",
         parameter="Power",
         fill=True,
-        color="blue",
+        color="#0061A4",
     )
     add_trace_to_figure(
-        df_sel, fig, component="Electrolyzer", parameter="Power", color="black"
+        df_sel, fig, component="Electrolyzer", parameter="Power", color="#0C0C0C"
     )
     add_trace_to_figure(
-        df_sel, fig, component="Derivate production", parameter="Power", color="red"
+        df_sel, fig, component="Derivate production", parameter="Power", color="#C3A5D3"

```

Comment:
I personally cannot see this color well in front of yellow.

Reply:
I changed it to dark green https://github.com/agoenergy/ptx-boa/pull/431/commits/175fbfbab0f67425dd33600b9f4920c967f58e66

We are limited to the agora color scheme though